test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -116,7 +116,10 @@
     <None Update="Managment\Payloads\ExpectedHttpExtensionsPayload.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Update="Microsoft.Azure.WebJobs.Script.WebHost.deps.json">
+    <None Update="DepsFiles\net8.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json">
```

Comment:
Instead of needing to know which one to use in the test, you could leverage msbuild to do that for you. Note in this example I am renaming the deps files to include TFM in their name, but on copy I transform back to the original name.

``` xml
<None Update="Microsoft.Azure.WebJobs.Script.WebHost.Net6.deps.json"
    CopyToOutputDirectory="Always"
    TargetPath="Microsoft.Azure.WebJobs.Script.WebHost.deps.json"
    Condition="'%(TargetFramework)' == 'net6.0" />
<None Update="Microsoft.Azure.WebJobs.Script.WebHost.Net8.deps.json"
    CopyToOutputDirectory="Always"
    TargetPath="Microsoft.Azure.WebJobs.Script.WebHost.deps.json"
    Condition="'%(TargetFramework)' == 'net8.0" />
```